packages/pigeon/lib/ast.dart

```diff
@@ -56,11 +57,14 @@ class Method extends Node {
   /// Specifies how handlers are dispatched with respect to threading.
   TaskQueueType taskQueueType;
 
+  /// Any attached docummentation comments on the method.
```

Comment:
I'd add to the docstring why this is a List.  Is this all the lines of the docstring?